src/bundle/Resources/views/fieldtypes/preview/content_fields.html.twig

```diff
@@ -364,15 +364,31 @@
                 <td>{{ field.value.fileName }}</td>
             </tr>
             <tr>
-                <td>{{ 'ezimage.size'|trans|desc('Size') }}</td>
+                <td>{{ 'ezimage.size'|trans|desc('Size') }}:</td>
                 <td>{{ field.value.fileSize|ez_file_size( 1 ) }}</td>
             </tr>
             <tr>
-                <td>{{ 'ezimage.alternative_text'|trans|desc('Alternative text') }}</td>
+                <td>{{ 'ezimage.alternative_text'|trans|desc('Alternative text') }}:</td>
                 <td>{{ field.value.alternativeText }}</td>
             </tr>
+            <tr>
+                <td>{{ 'ezimage.master_dimensions'|trans|desc('Master dimensions') }}:</td>
+                <td>{{ field.value.width ~ 'px ' ~ 'ezimage.width'|trans|desc('Width') ~ ' - ' ~ field.value.height ~ 'px ' ~ 'ezimage.height'|trans|desc('Height') }}</td>
```

Comment:
I think it should be a one string with placeholders for width and height values. Is it possible?

Reply:
For me, it's perfectly fine. Translators will have full freedom to display data the way they want to.

Reply:
done :)